### PR TITLE
Cloud v24.2.0_cycle26 changelog

### PR DESCRIPTION
**Pull request by llewellyn-sl**

### CODE REVIEW

File: changelog/seqera-cloud/v24.2.0_cycle26.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+**Users that sign in with multiple accounts associated with the same email address** — You will be prompted to optionally merge your accounts into a single account. This will merge all your work across the two different accounts into the same account and give you the option to sign in via either method of authentication.  
+
+If you choose not to merge accounts, you will not be prompted again. If you wish to merge accounts later, or would like to merge accounts with different email addresses, contact [Seqera Support](https://support.seqera.io).  
+
+### Feature updates and improvements
+
+- Data Studios now support GPUs
+- The pipeline parameters input form now supports JSON schema draft 2020-12
+- Augment Data Studios images with custom Conda environment
+- Cookie authentication is now supported with auth0
```
Comment (swampie) [inline changelog/seqera-cloud/v24.2.0_cycle26.mdx:28]: this can be removed

File: changelog/seqera-cloud/v24.2.0_cycle26.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- Auth0 login now redirects after 10s
```
Comment (swampie) [inline changelog/seqera-cloud/v24.2.0_cycle26.mdx:29]: this can be removed

File: changelog/seqera-cloud/v24.2.0_cycle26.mdx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+- Data studio creation now includes YAML validation
+- Data Studios build stats are now included in the Dashboard
+- New data studios now include "Add only" and "Add and start" actions
+- MemStore implementation based on Jedis
```
Comment (swampie) [inline changelog/seqera-cloud/v24.2.0_cycle26.mdx:33]: this can be removed

File: changelog/seqera-cloud/v24.2.0_cycle26.mdx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+### Bug fixes 
+
+- Add back to login button in authentication error screen
+- Add custom token cookie login handler for Nextflow requests to receive authentication cookies
+- Data Studios UI fixes
+- Label regression
+- Visibility regression
+- Data Studios form validity is now evaluated from the start
```
Comment (swampie) [inline changelog/seqera-cloud/v24.2.0_cycle26.mdx:42]: remove

File: changelog/seqera-cloud/v24.2.0_cycle26.mdx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+- Jedis connection pool config which caused potential instabilities
+- Improve Data Studios error messaging 
+- Delete data studio checkpoint on build failure
+- Update default Data Studios tool versions
+- Failed data studios can now be deleted 
+- Increase data studio Conda environment textarea size
```
Comment (swampie) [inline changelog/seqera-cloud/v24.2.0_cycle26.mdx:48]: remove

File: changelog/seqera-cloud/v24.2.0_cycle26.mdx [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+- Jedis connection pool config which caused potential instabilities
+- Improve Data Studios error messaging 
+- Delete data studio checkpoint on build failure
+- Update default Data Studios tool versions
+- Failed data studios can now be deleted 
+- Increase data studio Conda environment textarea size 
+- Set launch form to parameters step view by default when a pipeline schema is present, regardless of specification
+- Dashboard persisted filters exclude workspace IDs the user no longer has access to
+- Do not fetch info for "INVALID" status pipelines, fetch for all other statuses including "DISABLED"
+- No class error when creating Azure Entra credentials
```
Comment (swampie) [inline changelog/seqera-cloud/v24.2.0_cycle26.mdx:52]: "Fixed a problem while creating new Azure Entra credentials"

File: changelog/seqera-cloud/v24.2.0_cycle26.mdx [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+- Return an empty list of data studios when receiving an invalid status
+- Fallback to primary compute environment when launching a shared pipeline without an associated compute environment from a private workspace
+- Initialize schema data with pipeline value from control
```
Comment (swampie) [inline changelog/seqera-cloud/v24.2.0_cycle26.mdx:55]: remove

**Review by drewdipalma — APPROVED** [review]
Approved pending unbolding the body text in the Auth release changelog entry

**Pull request merged**